application/routes/api.php

```diff
@@ -33,3 +34,13 @@
 Route::get('/jwt-claims', [JwtClaimsController::class, 'show'])->withoutMiddleware('auth:api');
 
 Route::get('/institutions', [InstitutionController::class, 'index']);
+
+Route::get(
+    '/institution-users/{institutionUserId}',
```

Comment:
**[non-blocking]** @DanBondarenko You defined query params names with camel case here but in other codebase parts we use the underscore, I think that it would be good to follow the same approach 🙂

Reply:
Yes, better to follow a consistent approach – changed it in latest revision